gradle.properties

```diff
@@ -15,5 +15,5 @@ org.gradle.jvmargs=-Xmx4096m
 # use parallel execution
 org.gradle.parallel=true
 
-android.enableUnitTestBinaryResources=true
 android.enableR8.fullMode=true
+android.useAndroidX=true
```

Comment:
for some reason this flag is required again

Reply:
I wanted to say that I use no such flags in my project, I checked it and I actually have this flag. Whatever, lol